dwitter/static/main.css

```diff
@@ -196,6 +196,10 @@ ul.messages li.success{
   word-wrap: break-word;
 }
 
+.comment-message code {
+  background: #eee;
```

Comment:
Might want to change font too?

Reply:
The font is already set by the top-level `code` CSS rule. Same font as the editor. Might want to play around with adding a border + padding around, but then layout might get thrown off.

Reply:
Oh, right. Looks pretty good already :)